examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-bed.prm

```diff
@@ -0,0 +1,298 @@
+# Listing of Parameters
+#----------------------
+
+set dimension = 3
+
+#---------------------------------------------------
+# Simulation Control
+#---------------------------------------------------
+
+subsection simulation control
+  set method               = bdf1
+  set number mesh adapt    = 0
+  set output name          = result
+  set output frequency     = 10
+  set startup time scaling = 0.6
+  set time end             = 5
+  set time step            = 0.001
+  set subdivision          = 1
+  set log precision        = 10
+  set output path          = ./output/
+end
+
+#---------------------------------------------------
+# Restart
+#---------------------------------------------------
+
+subsection restart
+  set checkpoint = true
+  set frequency  = 100
+  set restart    = false
+  set filename   = case
+end
+
+#---------------------------------------------------
+# FEM
+#---------------------------------------------------
+
+subsection FEM
```

Comment:
I would try the simulation with Q2-Q2 and compare the results, would be interesting.